scripts/wfl/twisttcap/setup_new_wfl_module.py

```diff
@@ -0,0 +1,220 @@
+import argparse
+import json
+import requests
+import google.auth as googleauth
+from firecloud import api as fapi
+from time import sleep
+from pprint import pprint
+
+# DEVELOPER: update this field anytime you make a new docker image
+docker_version = "1.0"
+
+
```

Comment:
Another FYI that's probably not very applicable to this file on using [Python type hints](https://www.pythontutorial.net/python-basics/python-type-hints), which becomes more and more valuable as Python modules get more complicated and interconnected although the cut-off would probably vary considerably from software eng to software eng 😊

Reply:
ah I'd love to start doing this. would you recommend using `mypy` like that article suggests?

Reply:
`mypy` is what I used but it was over 3 years ago so not sure if there may be a better tool now but probably no need to make perfect the enemy of good even if there is 😊